mono_repo.yaml

```diff
@@ -0,0 +1,23 @@
+# Created by mono_repo
```

Comment:
Does the auto-detect logic not work here? Ideally, this file shouldn't be needed...

Reply:
Ah, ok. Wasn't sure what the best practice was for mono_repo. If we should check this in or not...
Guess the minor optimization of having this info cached is not worth the potential for stale data.